test/verify/check-system-services

```diff
@@ -369,31 +382,34 @@ Unit=test.service
 
         # Select only static services
         init_filter_state()
-        b.select_from_dropdown("#services-dropdown-file-state", "static")
+        self.select_file_state("Static")
         self.wait_service_not_present("test.service")
         self.wait_service_not_present("test-fail.service")
         self.wait_service_present("systemd-halt.service")
 
         # Select only disabled services
-        b.select_from_dropdown("#services-dropdown-file-state", "disabled")
+        init_filter_state()
+        self.select_file_state("Disabled")
         self.wait_service_present("test.service")
         self.wait_service_not_present("test-fail.service")
         self.wait_service_not_present("systemd-halt.service")
 
         # Select only stopped services
         init_filter_state()
-        b.select_from_dropdown("#services-dropdown-active-state", "inactive")
+        self.select_active_state("Not running")
         self.wait_service_present("test.service")
         self.wait_service_not_present("test-fail.service")
 
         # Select only failed services
-        b.select_from_dropdown("#services-dropdown-active-state", "failed")
+        init_filter_state()
+        self.select_active_state("Failed to start")
         self.wait_service_not_present("test.service")
         self.wait_service_present("test-fail.service")
 
         # Check filtering and selecting together - empty state
+        init_filter_state()
```

Comment:
Can you please also replace one of these to check that those `x` for removing it from UI work?

Reply:
Ideally in both ways - X to remove all of one group and X to remove just one specific item.

Reply:
Done